member.go

```diff
@@ -123,6 +123,8 @@ type UpdateMember struct {
 	Mute      *bool        `json:"mute,omitempty"`
 	Deaf      *bool        `json:"deaf,omitempty"`
 	ChannelID *Snowflake   `json:"channel_id,omitempty"`
+	// CommunicationDisabledUntil defines when the user's timeout will expire and the user will be able to communicate in the guild again (up to 28 days in the future)
+	CommunicationDisabledUntil *Time `json:"communication_disabled_until,omitempty"`
```

Comment:
One caveat is as we use `omitempty`, you can't set this field to `null` from what I can tell to remove a users timeout. Perhaps the `Time` struct could be extended to support returning `null` in `MarshalJSON` when you set something like `Time.Null = true` instead of returning an empty string if you set the time to zero: https://github.com/andersfylling/disgord/blob/master/struct.go#L85-L88

Reply:
Do you want to throw up a tests to show this behavior? A simple unit table tests or something.

Reply:
I added a unit test to show it as `TestUpdateMemberParams` 👍